test/tests/parallel/set1/spam_virus/rspamd_full.bats

```diff
@@ -45,16 +45,16 @@ function setup_file() {
 
   # We will send 4 emails:
   #   1. The first one should pass just fine
-  _send_email_and_get_id MAIL_ID_PASS
+  _send_email_with_msgid 'pass'
```

Comment:
All the `MSG_ID` header values provided in this test file could probably use a more grep friendly value, especially since `virus` / `header` / `pass` / `spam` are all very generic 😬

Reply:
👍🏼

Reply:
Done in 143ca0c27d686c44882b7e045e1c98574a20995d